solidity/dashboard/src/sagas/subscriptions.js

```diff
@@ -809,3 +857,26 @@ export function* subscribeToLiquidityRewardsEvents() {
     )
   }
 }
+
+export function* watchLiquidityRewardNotifications() {
+  const {
+    eth: { defaultAccount },
+  } = yield getWeb3Context()
+
+  // for the first iteration update the lastNotificationRewardAmount variable in redux without showing message
+  let displayMessage = false
+  while (true) {
+    for (const pairName of Object.keys(LIQUIDITY_REWARD_PAIRS)) {
+      yield put({
```

Comment:
I think we can use `yield fork(processLiquidityRewardEarnedNotification, action)` so we can get rid of an additional `watchLiquidityRewardsEarnedNotification` watcher in `sagas/liquidity-rewards`.